vulcanai2/models/basenetwork.py

```diff
@@ -141,9 +140,12 @@ def __init__(self, name, config, in_dim=None, save_path=None,
         self.out_dim = self._get_out_dim()
 
         self.device = device
+        self._optim_spec = optim_spec
         self._criter_spec = criter_spec
+        self.optim = None
```

Comment:
This was much debated before, you're just adding it back in so that all class parameters are included in init? nothing else changes/needs to change?